HOW/_requirements.md

```diff
@@ -0,0 +1,30 @@
+---
+HOW: <path in HOW tree>
+Source: <path to process used to create this HOW>
+Created: <YYYY-MM-DD>
+Version: <version-type(semantic/index)/version-number>
+Status: <initial status from process>
+---
+
+## Title
+The title of the standard being defined
+
+## Author(s)
+
+list of authors' names and optionally, email addresses
+
+## Abstract
+
+A short (~200 word) description of the technical/social/informational issue being addressed.
+
+## Copyright
+
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
+## Motivation
+
+Clearly explain why this standard is being proposed. If the standard is a new version of an existing standard, or makes one SUPERSEDED explain why that is inadequate to address the problem that this HOW solves. Include specific use cases and describe why this HOW is valuable to the Holochain ecosystem.
```

Comment:
> that this HOW solves

What does HOW stand for? Should this read `Holochain proposal" or something like that?

Reply:
It currently doesn't stand for anything. A few ideas have been tossed out (I like Holochain Organized Wisdom). I would prefer to pick something for it to stand for than to have it stand for nothing.

Reply:
I agree. I'd rather change it to something common in the software engineering industry than using a term (that's also a common word like "how") that we need to invent a meaning for.